examples/basic/lines/arcs.py

```diff
@@ -1,3 +1,14 @@
+''' A arc graph using pre-defined data points. This example
+demonstrates the use of the ``arc`` method to make a graph
+by drawing three arcs of defined radius, defined start and
+end angles at defined point(coordinates).
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.arc
+    :refs: :ref:`ug_basic_lines_arcs
+    :keywords: arc, figure, glyph
+
+'''
```

Comment:
@KesharwaniArpita did you intend to combine the two previous PRs into one? If so that's fine but please update the PR title to match.